<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: white;
	border: 1px solid #D9D9D9;
	border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>202</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kode Barang</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addItemCode">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 1px solid #D9D9D9;
	border-radius: 6px;
	padding:0 10px 0 10px;
}

QLineEdit:focus{
	border: 1px solid #3D6CEE;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: white;
	border: 1px solid #D9D9D9;
	border-radius: 6px;
}

QPushButton:hover{
	color: #3D6CEE;
	border: 1px solid #3D6CEE;
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>432</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Keterangan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addDescription">
      <property name="minimumSize">
       <size>
        <width>430</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 1px solid #D9D9D9;
	border-radius: 6px;
	padding:0 10px 0 10px;
}

QLineEdit:focus{
	border: 1px solid #3D6CEE;
}</string>
      </property>
      <property name="maxLength">
       <number>430</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>202</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Jumlah</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addAmount">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 1px solid #D9D9D9;
	border-radius: 6px;
	padding:0 10px 0 10px;
}

QLineEdit:focus{
	border: 1px solid #3D6CEE;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>5</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border: none;
}

QPushButton:hover{
	border-left: 1px solid #D9D9D9;
	border-bottom: 1px solid #D9D9D9;
	border-bottom-left-radius: 6px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/close-blue.svg</normaloff>:/icons/close-blue.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>202</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Jenis</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addType">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 1px solid #D9D9D9;
	border-radius: 6px;
	padding:0 10px 0 10px;
}

QLineEdit:focus{
	border: 1px solid #3D6CEE;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: white;
	border: 1px solid #D9D9D9;
	border-radius: 6px;
}

QPushButton:hover{
	color: #3D6CEE;
	border: 1px solid #3D6CEE;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>202</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tanggal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addDate">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 1px solid #D9D9D9;
	border-radius: 6px;
	padding:0 10px 0 10px;
}

QLineEdit:focus{
	border: 1px solid #3D6CEE;
}</string>
      </property>
      <property name="placeholderText">
       <string>yyyy-mm-dd</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>243</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add Transaction</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/icons/transactions-blue.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../assets/icons/resources.qrc"/>
  <include location="../../../../Python/Lib/site-packages/qt5_applications/Qt/assets/icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
